Exiled.API/Features/Door.cs

```diff
@@ -372,6 +372,7 @@ public static void UnlockAll(Func<Door, bool> predicate)
         /// <summary>
         /// Breaks the specified door. No effect if the door cannot be broken, or if it is already broken.
         /// </summary>
+        /// <param name="type">The <see cref="DoorDamageType"/> to apply to the door.</param>
         /// <returns><see langword="true"/> if the door was broken, <see langword="false"/> if it was unable to be broken, or was already broken before.</returns>
         [Obsolete("BreakDoor() will be obsolete in future versions, please use BreakDoor(DoorDamageType)", false)]
         public bool BreakDoor() => BreakDoor(DoorDamageType.ServerCommand);
```

Comment:
Ew, why?

Reply:
I didn't even make this change wtf, look at changed files